system-variables.md

```diff
@@ -320,85 +400,91 @@ Constraint checking is always performed in place for pessimistic transactions (d
+### tidb_enable_rate_limit_action
+
+- Scope: SESSION | GLOBAL
+- Default value: ON
+- This variable controls whether to enable the dynamic memory control feature for the operator that reads data. By default, this operator enables the maximum number of threads that [`tidb_disql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) allows to read data. When the memory usage of a single SQL statement exceeds [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) each time, the operator that reads data stops one thread.
+- When the operator that reads data has only one thread left and the memory usage of a single SQL statement continues to exceed [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query), this SQL statement triggers other memory control behaviors, such as [spilling data to disk](/tidb-configuration-file.md#spilled-file-encryption-method).
 
 ### tidb_enable_slow_log
 
 - Scope: INSTANCE
-- Default value: `1`
-- This variable is used to control whether to enable the slow log feature. It is enabled by default.
+- Default value: ON
+- This variable is used to control whether to enable the slow log feature.
 
 ### tidb_enable_stmt_summary <span class="version-mark">New in v3.0.4</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: 1 (the value of the default configuration file)
+- Default value: ON (the value of the default configuration file)
 - This variable is used to control whether to enable the statement summary feature. If enabled, SQL execution information like time consumption is recorded to the `information_schema.STATEMENTS_SUMMARY` system table to identify and troubleshoot SQL performance issues.
 
 ### tidb_enable_table_partition
 
 - Scope: SESSION | GLOBAL
-- Default value: "on"
-- This variable is used to set whether to enable the `TABLE PARTITION` feature.
-
-    - `off` indicates disabling the `TABLE PARTITION` feature. In this case, the syntax that creates a partition table can be executed, but the table created is not a partitioned one.
-    - `on` indicates enabling the `TABLE PARTITION` feature for the supported partition types. Currently, it indicates enabling range partition, hash partition and range column partition with one single column.
-    - `auto` functions the same way as `on` does.
+- Default value: ON
+- This variable is used to set whether to enable the `TABLE PARTITION` feature:
 
-- Currently, TiDB only supports Range partition and Hash partition.
+    - `ON` indicates enabling Range partitioning, Hash partitioning, and Range column partitioning with one single column.
+    - `AUTO` functions the same way as `ON` does.
+    - `NIGHTLY` indicates enabling the types of partitioned tables specified in `ON`. It also indicates enabling List partitioning and List COLUMNS partitioning.
+    - `OFF` indicates disabling the `TABLE PARTITION` feature. In this case, the syntax that creates a partition table can be executed, but the table created is not a partitioned one.
```

Comment:
List columns feature is not released with TiDB 5.0 RC. So this doc change is reverted in https://github.com/pingcap/docs/pull/4593/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R374-R378.

Reply:
Can we merge to master and then remove from 5.0-rc?

Reply:
The content of List partition/List columns partition is also removed from `master`. 
```suggestion
    - `ON` indicates enabling Range partitioning, Hash partitioning, and Range column partitioning with one single column.
    - `AUTO` functions the same way as `ON` does.
    - `OFF` indicates disabling the `TABLE PARTITION` feature. In this case, the syntax that creates a partition table can be executed, but the table created is not a partitioned one.
```